docs/tutorials/integrations/kserve/kserve-http.md

```diff
@@ -4,7 +4,7 @@ template: main.html
 
 # Load Test a KServe Model (via HTTP)
 
-This tutorial shows how easy it is to run a load test for KServe when using HTTP to make requests. We use an scikit-learn model to demonstrate. The same approach works for any model type. 
+This tutorial shows how easy it is to run a load test for KServe when using HTTP to make requests. We use a scikit-learn model to demonstrate. The same approach works for any model type. 
```

Comment:
In the sentence below:

> You can assert experiment outcomes, view an experiment report, and view experiment logs as described in your first experiment.

Could you please link to [your first experiment](link).